src/pages/games/midnight-train.astro

```diff
@@ -0,0 +1,328 @@
+---
```

Comment:
The Midnight Train page is mostly just ported-over, but we did lose the navigation menu in the process. Shouldn't be too hard to copy over as well (I was just lazy), and we can implement it as a component now instead.